tests/integration/cli/repos_test.py

```diff
@@ -0,0 +1,69 @@
+from os import path
+import shutil
+from subprocess import check_output
+
+import git
+
+from dusty.constants import REPOS_DIR
```

Comment:
Should probably override this to a temp dir like we do in a few other places. As long as we do module-level imports everywhere, should get picked up fine.